content/documentation/_index.md

```diff
@@ -31,15 +32,15 @@ Welcome to the IPFS Cluster documentation. This is where you will find user docu
 
 ## User guides
 
-* [Quickstart: Starting, enlarging and shrinking a cluster](/guides/quickstart)
+* [Quickstart: Starting, enlarging and shrinking a cluster](quickstart)
```

Comment:
please don't change urls. This will break links that we have posted before to the world.

Reply:
Just checked this one (as per the comment below). Redirect works so it will not break wild links.